Frends.AzureEventHub.Receive/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [2.0.2] - 2024-05-31
```

Comment:
We had previously a not so good version bumping schema, but lets start fixing that in this task as well - lets bump second version number, so the new version here would be 2.1.0, instead of 2.0.2